packages/sdk-react/src/core/queries/useReceivables.ts

```diff
@@ -34,6 +34,16 @@ const receivablesQueryKeys = {
   detail: (id: string) => [...receivablesQueryKeys.all(), 'detail', id],
 };
 
+const useReceivablePdfListCache = () => {
+  const queryClient = useQueryClient();
+
+  return {
+    invalidate: (receivableId: string) => {
+      queryClient.setQueryData(receivablesQueryKeys.pdf(receivableId), null);
+    },
+  };
```

Comment:
I think the previous implementation was better because it would be easier to fix in the future.

Reply:
Moved it back [289e24e](https://github.com/team-monite/monite-sdk/pull/87/commits/289e24e46beae79fbab15a4d0d0b167deed7b20b)

Reply:
[c1e18c8](https://github.com/team-monite/monite-sdk/pull/87/commits/c1e18c8e7e7434340e198d3b482c35ef15e7d9e6)